src/rgw/rgw_file.cc

```diff
@@ -1944,7 +1945,8 @@ namespace rgw {
     real_time appx_t = real_clock::now();
 
     state->obj_size = bytes_written;
-    perfcounter->inc(l_rgw_put_b, state->obj_size);
+    auto counters = rgw::op_counters::get(state);
+    rgw::op_counters::inc(counters, l_rgw_op_put_b, state->obj_size);
```

Comment:
in rgw_file.cc, these calls to `inc()`/`tinc()` are all happening in different functions of `RGWWriteRequest`. please stash the `CountersContainer` variable in `RGWWriteRequest` so we only have to call `get()` once in `RGWWriteRequest::exec_start()`